clients/drcachesim/common/options.h

```diff
@@ -53,10 +46,20 @@
 #define VIEW "view"
 #define FUNC_VIEW "func_view"
 #define INVARIANT_CHECKER "invariant_checker"
+#define SCHEDULE_STATS "schedule_stats"
+
+#define REPLACE_POLICY_NON_SPECIFIED ""
```

Comment:
Why was this moved below? If the goal was  to separate vs the tool name constants, wouldn't just an additional vertical whitespace suffice?

Also, add comment to mark the logical separation of the constants. "all tool name constants" etc.

Reply:
Two reasons: A) to group it with the other per-tool defines below (CACHE_TYPE_INSTRUCTION, etc.) and B) it seemed more logical to have the tool names first, and then have per-tool specific constants.  Added comments.